mono/metadata/remoting.c

```diff
@@ -1902,6 +1909,9 @@ mono_marshal_get_proxy_cancast (MonoClass *klass)
 	mb->method->save_lmf = 1;
 
 #ifndef DISABLE_JIT
+
```

Comment:
Just call mono_remoting_marshal_init () here like in the other cases ?

Reply:
yeah, I tried that. I had some failures, I could not easily explain, so I decided to try something semantically smaller.
You know, to skip the, I'm not sure what it is -- class loading?
I can try that again. I'll make a separate PR for that, so the success stands here in case that fails/is-rejected.